metadata/v1.8/step_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.8",
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.8",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```

Comment:
source code url here should be the source files you created together with this PR, but we can use another PR to update them

Reply:
By the way, this `source_code_url` seems to be auto-generated. `release.yaml` did have the field.

Reply:
I think the previous `source_code_url` was the default string. The original `release.yaml` didn't have the field.

Reply:
@sherry-x , I updated `source_code_url` to https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.8/step_1_upgrade_framework